test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -282,6 +288,34 @@ def test_emacs(image):
     _run_cmd_on_container(container_name, ctx, "emacs -version")
 
 
+@pytest.mark.model("N/A")
+@pytest.mark.integration("sagemaker python sdk")
+def test_sm_pysdk_2(tensorflow_training):
+    """
+    Simply verify that we have sagemaker > 2 in the python sdk.
+
+    If you find that this test is failing because sm pysdk version is not greater than 2, then that means that
+    the image under test needs to be updated.
+
+    If you find that the training image under test does not have sagemaker pysdk, it should be added or explicitly
+    skipped (with reasoning provided).
+
+    :param tensorflow_training: training ECR image URI - to be updated to simply "training" when mx/pt are added
+    """
+    image_uri = tensorflow_training
+
+    # Ensure that sm pysdk2 is on the container
+    ctx = Context()
+    container_name = _get_container_name("sm_pysdk", image_uri)
+    _start_container(container_name, image_uri, ctx)
+
+    sm_version = _run_cmd_on_container(
+        container_name, ctx, "import sagemaker; print(sagemaker.__version__)", warn=True, executable="python"
```

Comment:
Why does `warn` need to be `True`?

Reply:
nice catch, leftover from previous implementation. will remove.